src/app.js

```diff
@@ -149,13 +149,16 @@ export default async logger => {
       }
     )
     fastify.post(
-      `/:org(${SLUG_REGEXP_STR})/${SERVICE_1_1}`,
+      `/:org/${SERVICE_1_1}`,
```

Comment:
This can introduce issues without URL validation. Keep the regex but change it to open up the character set. Also rename from `SLUG_REGEXP_STR` since it's no longer a slug technically.

```
const ORG_REGEXP_STR = '^[A-Za-z][A-Za-z0-9-_]{2,}[A-Za-z0-9]$'
```